src/main/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParser.java

```diff
@@ -1050,6 +1055,9 @@ private OptionDefinition(final Field field, final String name, final String shor
             this.isCommon = isCommon;
             this.mutuallyExclusive = new HashSet<String>(Arrays.asList(mutuallyExclusive));
             this.printOrder = printOrder;
+            if (this.field.getAnnotation(Hidden.class) != null) {
+                logger.warn("Hidden argument --{} is not handled", this.name);
```

Comment:
Its the annotation thats not handled, not the argument. This should probably say something like "Hidden annotation is not honored for --{}".